src/main/java/org/itsallcode/io/Capturable.java

```diff
@@ -1,22 +1,25 @@
 package org.itsallcode.io;
 
+/**
+ * Interface for classes that can capture output.
+ */
 public interface Capturable
 {
     /**
      * Activate capturing
      */
-    public void capture();
+    void capture();
```

Comment:
Why remove this? I always found this notation inconsistent.

Reply:
Sonar complains about this. Interface methods are public by default.